packages/devtools_app/lib/src/charts/treemap.dart

```diff
@@ -647,7 +647,16 @@ class TreemapNode extends TreeNode<TreemapNode> {
         // the Dart AOT snapshot.
         return reversedPath.reversed.toList();
       }
-      if (current.name.contains('package:')) {
+
+      // Prevent duplicated package names from being added to the path.
+      // Sometimes the hierarchy can look like this:
+      // package:a
+      //     package: a
+      //     <Type>
+      //     <OneByteString>
+      if ((current.name.contains('package:') ||
```

Comment:
not a new issue but can we use startsWIth instead of contains to avoid false positives?

Reply:
done